editor/src/communication/dispatcher.rs

```diff
@@ -339,10 +339,19 @@ mod test {
 		init_logger();
 		let mut editor = create_editor_with_three_layers();
 
-		let sorted_layers = editor.dispatcher.documents_message_handler.active_document().all_layers_sorted();
+		fn to_vec<'a>(paths: Vec<&'a [LayerId]>) -> Vec<Vec<LayerId>> {
+			paths.iter().map(|layer| layer.to_vec()).collect::<Vec<_>>()
+		}
+		let sorted_layers = to_vec(editor.dispatcher.documents_message_handler.active_document().all_layers_sorted());
 		println!("Sorted layers: {:?}", sorted_layers);
```

Comment:
Probably should be debug / trace.

Reply:
Thought the same thing, but afaik this is only used for debugging test cases which use std out

Reply:
We could put it into a #[cfg(test)] though